src/index.spec.ts

```diff
@@ -1,35 +1,10 @@
+import * as INDEX from '@/index';
```

Comment:
kan deze naamgeving niet wat descriptiever?

Reply:
het is voor de test, ik wil alleen kijken wat die index exporteert